p3-weather-journal-app/website/app.js

```diff
@@ -3,3 +3,111 @@
 // Create a new date instance dynamically with JS
```

Comment:
replace all `let` and `var` with const

Reply:
done in: [20ecb96](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/7/commits/20ecb96ba6c5ac532cbc0ea394143deb379eb132)